docs/reference/pip.rst

```diff
@@ -90,6 +90,25 @@ before invoking ``setup.py``. The injection should be transparent to
 ``setup.py`` emulating the commands pip requires may need to be aware that it
 takes place.
 
+Build System Output
+~~~~~~~~~~~~~~~~~~~
+
+Any output produced by the build system will be read by pip (for display to the
+user if requested). In order to correctly read the build system output, pip
+requires that the output is written in a well-defined encoding, specifically
+the encoding the user has configured for text output (which can be obtained in
+Python using ``locale.getpreferredencoding``). If the configured encoding is
+ASCII, pip assumes UTF-8 (to match the behaviour of some Unix systems).
```

Comment:
OK. I'll go for "accounting for the behaviour of" (don't want to be accused of being the Windows guy criticising Unix ;-))